src/Build/Logging/ProfilerLogger.cs

```diff
@@ -271,17 +271,25 @@ private static bool IsTooSmall(ProfiledLocation profiledData)
         }
 
         /// <summary>
-        /// Gets the markdown content of the aggregated results and saves it to disk
+        /// Pretty prints the aggregated results and saves it to disk
         /// </summary>
+        /// <remarks>
+        /// If the extension of the file to log is 'md', markdown content is generated. Otherwise, it falls 
+        /// back to a tab separated format
+        /// </remarks>
         private void GenerateProfilerReport()
         {
             try
             {
-                var profilerFile = FileToLog;
-                Console.WriteLine(ResourceUtilities.FormatResourceString("WritingProfilerReport", profilerFile));
+                Console.WriteLine(ResourceUtilities.FormatResourceString("WritingProfilerReport", FileToLog));
```

Comment:
Oops, I missed these in previous PRs. Can you please use `Log.LogMessageFromResources` instead of console?

Reply:
As discussed offline, doing Console.WriteLine seems to be fine

---
In reply to: [159565737](https://github.com/Microsoft/msbuild/pull/2837#discussion_r159565737) [](ancestors = 159565737)

Reply:
Mistook the Logger for a task. Ignore :)